### PR TITLE
[TASK] Update "Create a custom content element type" for TYPO3 v13

### PR DESCRIPTION
**Pull request by linawolf**
The "Extended example" lines 244ff will be adjusted in a followup.

Depends on: https://github.com/TYPO3-Documentation/TYPO3CMS-Reference-TCA/pull/1165

Related: https://github.com/TYPO3-Documentation/t3docs-examples/pull/291/files
Related: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/1013
Related: https://github.com/TYPO3-Documentation/Changelog-To-Doc/issues/801
Releases: main

### CODE REVIEW

File: Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_tt_content.php [hunk 1]
```diff
@@ -1,29 +1,32 @@
...
-        'group' => 'default',
-        // description
-        'description' => 'LLL:EXT:my_extension/Resources/Private/Language/locallang.xlf:myextension_newcontentelement_description',
-    ],
-    'textmedia',
-    'after',
-);
+call_user_func(function () {
+
+    $key = 'examples_basiccontent';
```
Comment (froemken) [inline Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_tt_content.php:10]: This key does not match value `examples_newcontentelement` in documentation.

Quote: "We use examples_newcontentelement throughout the simple example."

File: Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst [hunk 2]
```diff
@@ -78,76 +78,82 @@ The following call needs to be added to the file
...
 
 .. index:: Content element; Icon
 .. _AddingCE-Icon:
 
-About the icon
---------------
+Display an icon
+---------------
+
+If you define no item a default item will be displayed.
```
Comment (froemken) [inline Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst:94]:
```suggestion
If you define no icon a default icon will be displayed.
```

File: Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst [hunk 3]
```diff
@@ -78,76 +78,82 @@ The following call needs to be added to the file
...
-    :caption: EXT:my_extension/Configuration/page.tsconfig
+The new content element wizard
+==============================
 
-.. versionchanged:: 12.0
+..  versionchanged:: 13.0
+    Starting with TYPO3 13.0 content elements added via TCA are automatically
+    displayed in the :guilabel:`New Content Element` wizard. To stay compatible
+    with both TYPO3 v12.4 and v13 keep the page TSconfig for TYPO3 v12.4.
+    See :ref:`New Content Element wizard in TYPo3 12.4 <t3coreapi/v12:AddingCE-PageTSconfig>`.
```
Comment (froemken) [inline Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst:112]:
```suggestion
    See :ref:`New Content Element wizard in TYPO3 12.4 <t3coreapi/v12:AddingCE-PageTSconfig>`.
```

File: Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst [hunk 4]
```diff
@@ -78,76 +78,82 @@ The following call needs to be added to the file
...
 
-   Starting with TYPO3 version 12.0 file :file:`EXT:examples/Configuration/page.tsconfig`
-   is automatically included. For version 11.5 and below this file has to be included in the
-   :file:`ext_localconf.php`. See :ref:`Setting global page TSconfig, compatible with TYPO3
-   11 and 12 <t3tsconfig:page-tsconfig-v11-v12>`.
+..  literalinclude:: _AddingYourOwnContentElements/_tt_content.php
+    :caption: EXT:my_extension/Configuration/TCA/Overrides/tt_content.php
+    :linenos:
+    :emphasize-lines: 16,17,18
```
Comment (froemken) [inline Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst:117]:
```suggestion
    :emphasize-lines: 17,18,19
```

File: Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst [hunk 5]
```diff
@@ -78,76 +78,82 @@ The following call needs to be added to the file
...
+
+Configure the backend fields for your new content element in
 the file :file:`Configuration/TCA/Overrides/tt_content.php`:
 
-..  literalinclude:: _AddingYourOwnContentElements/_tt_content_2.php
-    :language: php
+..  literalinclude:: _AddingYourOwnContentElements/_tt_content.php
     :caption: EXT:my_extension/Configuration/TCA/Overrides/tt_content.php
+    :linenos:
+    :emphasize-lines: 27,28
```
Comment (froemken) [inline Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst:149]:
```suggestion
    :emphasize-lines: 28,29
```

File: Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst [hunk 6]
```diff
@@ -78,76 +78,82 @@ The following call needs to be added to the file
...
 
-Now the backend form for the new content elements looks like this:
-
-.. include:: /Images/AutomaticScreenshots/CustomContentElements/ContentElementFields.rst.txt
+In line 27 a custom :ref:`palette <t3tca:palettes>` with the header and related
```
Comment (froemken) [inline Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst:151]:
```suggestion
In line 28 a custom :ref:`palette <t3tca:palettes>` with the header and related
```

File: Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst [hunk 7]
```diff
@@ -78,76 +78,82 @@ The following call needs to be added to the file
...
+fields is displayed. This palette and its fields are defined in
+:t3src:`typo3/sysext/frontend/Configuration/TCA/tt_content.php`.
 
+In line 28 a predefined field, `bodytext` is added to be displayed in the
```
Comment (froemken) [inline Documentation/ApiOverview/ContentElements/AddingYourOwnContentElements.rst:155]:
```suggestion
In line 29 a predefined field, `bodytext` is added to be displayed in the
```

File: Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_setup_2.typoscript [hunk 8]
```diff
@@ -1,6 +1,6 @@
 tt_content {
-  myextension_newcontentelement =< lib.contentElement
-  myextension_newcontentelement {
-    templateName = NewContentElement
+  examples_basiccontent =< lib.contentElement
```
Comment (froemken) [inline Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_setup_2.typoscript:2]: Does not match example key: `examples_newcontentelement`

File: Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_tt_content.php (hunk 1 above)
Comment (linawolf) [reply Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_tt_content.php]: I unified the key to myextension_basiccontent all over, as we also call the extension my_extension by default

File: Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_setup_2.typoscript (hunk 8 above)
Comment (linawolf) [reply Documentation/ApiOverview/ContentElements/_AddingYourOwnContentElements/_setup_2.typoscript]: Unified them

**Review by froemken — APPROVED** [review]

**Pull request merged**